### DISCUSSION TITLE
What where the specs of a system that had the Gravis Ultrasound?

### DISCUSSION THREAD

**Discussion by robman501a** [86Box/86Box#3294] [Q&A]
Any idea what the specs of a system where which has the Gravis Ultrasound? I want to try to emulate a system that used this sound card but I don't know how I should configure 86box.
Is the Ultrasound usually paired with another sound card (ie. SBpro, SB16) or usually ran as the only sound card in a system?

**Comment by TC1995** [comment]
The GUS can also be standalone.
